src/Cech_complex/include/gudhi/MEB_filtration.h

```diff
@@ -0,0 +1,112 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Marc Glisse
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef MEB_FILTRATION_H_
+#define MEB_FILTRATION_H_
+
+namespace Gudhi::cech_complex {
+
+/**
+ * \ingroup cech_complex
+ *
+ * \brief
+ * Given a simplicial complex and an embedding of its vertices, this assigns to
+ * each simplex a filtration value equal to the squared radius of its minimal
+ * enclosing ball.
+ *
+ * Applied on a Cech complex, it recomputes the same values (squared). Applied on a Delaunay triangulation, it computes the Delaunay-Cech filtration.
```

Comment:
Delaunay that derives from Alpha sounds strange, if anything I would expect the reverse (although if it is private inheritance, it could be ok).
Sure, you can open an issue about improving the visibility of Delaunay (through better doc or separate API or ...).

Reply:
cf. #967